content/20.results.md

```diff
@@ -70,16 +68,19 @@ Importantly, this model did not achieve as high an accuracy as the model trained
 ### Assessing the Nationality Diversity of Authors and Honorees
 
 We applied our Wiki2019-LSTM model to both our computational biology honorees dataset and our dataset of corresponding authors.
-We saw proportional discrepancies between the two groups, namely a large overrepresentation of CelticEnglish (including American) keynote speakers and a large underrepresentation of East Asian keynote speakers.
+We found that the proportion of authors in the CelticEnglish categories had decreased (Fig. {@fig:region_breakdown}A), particularly for papers published in _Bioinformatics_ and _BMC Bioinformatics_ (Fig. {@fig:region_breakdown}B).
+Among keynote speakers and fellows we found that the majority of honorees are predicted to be from CelticEnglish countries ({@fig:region_breakdown}C).
+Though sample sizes were small, we did observe some differences between meetings.
+ISMB keynotes had more probability attributable to Israel, while RECOMB had more attributable to EastAsian countries ({@fig:region_breakdown}D).
```

Comment:
I'm considering moving the busy panels to supplement, but I can put them back in if we think they're important for the main text.

Reply:
Ok - will go with this for now and then we can continue to revise and cite the supplementary figs instead if things move.